modules/data.atmosphere/R/download.PalEON_ENS.R

```diff
@@ -0,0 +1,62 @@
+##' Download PalEON files
+##'
+##' @name download.PalEON
+##' @title download.PalEON
```

Comment:
I would drop the `@name` tag. I think the only time that should be used is if you want the documentation name to be explicitly different from the function it's describing, and in this case, I don't think that's what you want.

The `@title` tag is the title of the documentation page. I think what you _actually_ want is to drop this line as well and stick the `@title` tag before `Download PalEON files`. Incidentally, the way the documentation is formatted now, the title is implied. By default, roxygen interprets blocks intelligently (though I think it's best to be explicit anyway), so a block like the following

```
#' Page title <Assumed to be the @title tag>
#' <blank>
#' Text <Assumed to be the @description tag>
#' @param, @author, etc...
```